python/setup.py

```diff
@@ -108,6 +108,7 @@ def run(self):
                       'namespace of boost (default: boost)'),
                      ('with-cuda', None, 'build the Cuda extension'),
                      ('with-flight', None, 'build the Flight extension'),
+                     ('with-hdfs', None, 'build the HDFS extension'),
```

Comment:
this seems unrelated?

Reply:
Removed (sorry, it bugs me when I have to remember which of env var/CLI argument to use for each component).